### ISSUE TITLE
Nie działa logowanie FB.

### ISSUE THREAD

**Issue by MaBober** [MaBober/SportoweSwiry#508]
@LukBartsch coś się wykrzyczało z logowaniem FB, może dałbyś radę tam zerknąć, bo to tak nie za ładnie wygląda jako portfolio :D

**Comment by LukBartsch** [comment]
@MaBober 
:O
Trzeba sprawdzić! 😉

**Comment by MaBober** [comment]
@LukBartsch reminderek, jakby Ci się udało rzucić okiem :)

**Comment by LukBartsch** [comment]
@MaBober bardzo dobrze ze przypominasz ;)

Jednak temat nie jest taki prosty.

Okazało się, że facebook zmienił jakiś czas temu swoją polityke w temacie aplikacji, ochrony danych itp. I obecnie żeby jakaś aplikacja mogła korzystać z funkcji logowania przez fb to należy wykonać weryfikacje takiej aplikacji. A polega to na powiązaniu aplikacji ze swoją firmą...

Próbowałem to jakoś zweryfikować na swoje dane, ale wszędzie krzyczy o przesłanie jakiś dokumentów potwierdzających posiadania firmy i chce wszystkie dane.

Wygląda na to, że bez posiadania firmy nie da sie tego używać publicznie. Na ten moment działa to tylko w trybie "developerskim" czyli można używać dla wybranych adresów email (administratorów aplikacji) - czyli tylko do testowania.

Szukam jakiegoś kanału wsparcia w meta/facebook. Ale wygląda na to ze nie ma nic takiego. Albo cholernie ciężko to znaleźć.

**Comment by MaBober** [comment]
> [@MaBober](https://github.com/MaBober) bardzo dobrze ze przypominasz ;)
> 
> Jednak temat nie jest taki prosty.
> 
> Okazało się, że facebook zmienił jakiś czas temu swoją polityke w temacie aplikacji, ochrony danych itp. I obecnie żeby jakaś aplikacja mogła korzystać z funkcji logowania przez fb to należy wykonać weryfikacje takiej aplikacji. A polega to na powiązaniu aplikacji ze swoją firmą...
> 
> Próbowałem to jakoś zweryfikować na swoje dane, ale wszędzie krzyczy o przesłanie jakiś dokumentów potwierdzających posiadania firmy i chce wszystkie dane.
> 
> Wygląda na to, że bez posiadania firmy nie da sie tego używać publicznie. Na ten moment działa to tylko w trybie "developerskim" czyli można używać dla wybranych adresów email (administratorów aplikacji) - czyli tylko do testowania.
> 
> Szukam jakiegoś kanału wsparcia w meta/facebook. Ale wygląda na to ze nie ma nic takiego. Albo cholernie ciężko to znaleźć.

Tak, wsparcie w Meta to inny poziom gęstości :S To proponuję chociaż na ten moment ukryć tą możliwość, w przyszłości można by się zastanowić, czy nie dałoby rady "podpiąć" tego np. pod JDG Kaśki.

**Comment by LukBartsch** [comment]
> > [@MaBober](https://github.com/MaBober) bardzo dobrze ze przypominasz ;)
> > Jednak temat nie jest taki prosty.
> > Okazało się, że facebook zmienił jakiś czas temu swoją polityke w temacie aplikacji, ochrony danych itp. I obecnie żeby jakaś aplikacja mogła korzystać z funkcji logowania przez fb to należy wykonać weryfikacje takiej aplikacji. A polega to na powiązaniu aplikacji ze swoją firmą...
> > Próbowałem to jakoś zweryfikować na swoje dane, ale wszędzie krzyczy o przesłanie jakiś dokumentów potwierdzających posiadania firmy i chce wszystkie dane.
> > Wygląda na to, że bez posiadania firmy nie da sie tego używać publicznie. Na ten moment działa to tylko w trybie "developerskim" czyli można używać dla wybranych adresów email (administratorów aplikacji) - czyli tylko do testowania.
> > Szukam jakiegoś kanału wsparcia w meta/facebook. Ale wygląda na to ze nie ma nic takiego. Albo cholernie ciężko to znaleźć.
> 
> Tak, wsparcie w Meta to inny poziom gęstości :S To proponuję chociaż na ten moment ukryć tą możliwość, w przyszłości można by się zastanowić, czy nie dałoby rady "podpiąć" tego np. pod JDG Kaśki.

Ukryć w sensie całkiem wywalić? czy jakoś po prostu zrobić "disable" ?

**Comment by MaBober** [comment]
> > > [@MaBober](https://github.com/MaBober) bardzo dobrze ze przypominasz ;)
> > > Jednak temat nie jest taki prosty.
> > > Okazało się, że facebook zmienił jakiś czas temu swoją polityke w temacie aplikacji, ochrony danych itp. I obecnie żeby jakaś aplikacja mogła korzystać z funkcji logowania przez fb to należy wykonać weryfikacje takiej aplikacji. A polega to na powiązaniu aplikacji ze swoją firmą...
> > > Próbowałem to jakoś zweryfikować na swoje dane, ale wszędzie krzyczy o przesłanie jakiś dokumentów potwierdzających posiadania firmy i chce wszystkie dane.
> > > Wygląda na to, że bez posiadania firmy nie da sie tego używać publicznie. Na ten moment działa to tylko w trybie "developerskim" czyli można używać dla wybranych adresów email (administratorów aplikacji) - czyli tylko do testowania.
> > > Szukam jakiegoś kanału wsparcia w meta/facebook. Ale wygląda na to ze nie ma nic takiego. Albo cholernie ciężko to znaleźć.
> > 
> > 
> > Tak, wsparcie w Meta to inny poziom gęstości :S To proponuję chociaż na ten moment ukryć tą możliwość, w przyszłości można by się zastanowić, czy nie dałoby rady "podpiąć" tego np. pod JDG Kaśki.
> 
> Ukryć w sensie całkiem wywalić? czy jakoś po prostu zrobić "disable" ?

Sorki, nie zauażyłem odpowiedzi. Usunąć przycisk moim zdaniem po prostu.
